Section offerings

natalka-hlushchuk opened the pull request:
Если фон задан контейнеру на промежутке 321-479 есть скачек
В коментах в scss offerings есть вариант бекграунда для секции, но он плавающий и там не проставлены варианты ретины

natalka-hlushchuk committed:
Section offerings

Yulenyka reviewed (changes requested):
Давай для картинок в папці  images создамо окрему папку offerings  и перенесемо туди усі твої картинки. Бо я думаю ще знайдуться люди котрі назвуть картинки так як ти. Ну і путі в css теж потрібно буде виправити.

Yulenyka reviewed (changes requested):
**!!! Пока не делать в понедельник уточним тянется он или нет**

1. // Підпривити Варіант зі фоном по контейнеру (скачок буде, але це нормально бо картинка зміюється)
.container--offerings {
  background-position: top center;
  background-repeat: no-repeat;
  background-size: cover;
  @media screen and (max-width: 320px) {
    background-image: url(../images/mobbg@1x.png);
  }
  @media screen and (min-width: 321px) and (max-width: 479px) {
    background-image: url(../images/mobbg480@1x.png);
  }
  @media screen and (max-width: $mob) {
    background-image: url(../images/mobbg480@1x.png);
  }
  @media screen and (min-width: $tab) {
    background-image: url(../images/tabbg@1x.png);
  }
  @media screen and (min-width: $desk) {
    background-image: url(../images/deskbg@1x.png);
    display: flex;
    gap: 90px;
    align-items: center;
  }
}
2. Огорнути два діва (яблоки і текст з кнопками) в один дів і цей новий дів падінгами відштовхнути зверху і знизу, і перенести флекси для десктопу.

Yulenyka reviewed (changes requested):
Кнопка повинная бути розміром 106 на 40
![Знімок екрана 2022-09-18 о 13 10 35](https://user-images.githubusercontent.com/55148574/190896963-5f42645d-6436-43ef-aa05-45e67149ed5a.png)

Yulenyka reviewed (changes requested):
**Планшет**
![Знімок екрана 2022-09-18 о 14 42 05](https://user-images.githubusercontent.com/55148574/190900288-97ff8e96-5263-41df-aefb-2138b3f13196.png)

- [ ] В планшеті у всіх твоїх  p   повинен бути  line-height: 1.14;

- [ ] у h2 line-height: 1.16;
- [ ] від тексту до кнопок відступ повинен бути 40px;

Yulenyka reviewed (changes requested):
**Десктоп**

- [ ] h2 -  повинен бути line-height: 1.19;
- [ ] h3 - перевір всюди, я йому стилі не завала, і ти теж)
- [ ] відступ від кнопок до тексту теж більший, як і в планшеті, підказка там 28 пікселів ти не туди прописала)

natalka-hlushchuk committed:
Внесені зміни після ревізії

1. Створена окрема папка для картинок
2. Доданий додатковий div "offerings__position
3. Відкореговані розміри кнопки
4. Додані стилі для h3
